src/DIRAC/ConfigurationSystem/Client/Utilities.py

```diff
@@ -139,12 +140,14 @@ def getGridCEs(vo, bdiiInfo=None, ceBlackList=None, hostURL=None, glue2=False):
     return result
 
 
-def getSiteUpdates(vo, bdiiInfo=None, log=None, glue2=True):
+def getSiteUpdates(vo, bdiiInfo=None, log=None, glue2=True, onecore=False):
     """Get all the necessary updates for the already defined sites and CEs
 
     :param str vo: VO name
     :param dict bdiiInfo: information from DBII
     :param object log: logger
+    :param bool glue2: whether to use glue2 information
+    :param bool onecore: whether to add single core copies of multicore queues
```

Comment:
I would link the github issue here, as a concrete example.

Reply:
I am going to refer to the Bdii2CSAgent Configuration for details, OK?